### PR TITLE
User Register and Login Implemented

### PR DESCRIPTION
**Pull request by jmartinacu**

### CODE REVIEW

File: routejet/core/models.py
Comment (jmartinacu) [inline routejet/core/models.py:1]: The creation of this new user, create a conflict with all the developers who have already created the app database. And for our app we don't need a specific implementation for our users.

**Review by jmartinacu — COMMENTED** [review]
The creation of this new user, create a dependencie conflict with all the developers who have already created the app database. And for our app we don't need a specific implementation for our users model.

**Comment by mohanadabulatifa** [comment]
everything correct to be merged

**Review by mohanadabulatifa — COMMENTED** [review]
every thing correct

**Review by mohanadabulatifa — APPROVED** [review]

**Pull request merged**